src/coreclr/jit/typelist.h

```diff
@@ -4,6 +4,9 @@
 #define GCS EA_GCREF
 #define BRS EA_BYREF
 #define EPS EA_PTRSIZE
+#ifdef TARGET_ARM64
+#define EAS EA_SCALABLE
```

Comment:
Similarly, I wonder if `SCALABLE` is the right term here. 

It would seem like the existing `EA_UNKNOWN` might be sufficient and would trigger relevant asserts in places that aren't correctly equipped to handle it. Alternatively, a different term might be a better fit